src/main/java/seedu/taskman/model/event/Frequency.java

```diff
@@ -1,21 +1,24 @@
 package seedu.taskman.model.event;
 
+import org.ocpsoft.prettytime.PrettyTime;
 import com.google.common.base.Objects;
-import org.apache.commons.lang.time.DurationFormatUtils;
+
 import seedu.taskman.commons.exceptions.IllegalValueException;
 import seedu.taskman.logic.parser.DateTimeParser;
 
-import java.time.Duration;
+import java.sql.Date;
 import java.time.Instant;
 
 public class Frequency {
-    // TODO: check for overflow because current validation allows years?
     public static final String MESSAGE_FREQUENCY_CONSTRAINTS =
             "Task frequency should only contain frequency and unit of time in the format: ";
+    // TODO: What is this differed thing?
     // differed: swap to multiple duration format
     public static final String FREQUENCY_VALIDATION_REGEX = "^" + DateTimeParser.SINGLE_DURATION + "$";
+    public static final int MULTIPLIER_TIME_UNIX_TO_JAVA = 1000;
```

Comment:
Shift to the function where its used, so it becomes a local variable, rather than a global one.